nginx/config/base.go

```diff
@@ -17,8 +17,13 @@ package config
 const BaseTemplate = `
 worker_processes 4;
 worker_rlimit_nofile 4096;
+{{if .is_windows}}
+pid C:/tmp/nginx.pid;
+{{else}}
 pid /tmp/nginx.pid;
+{{end}}
 user root root;
```

Comment:
does `user root root` work on windows? Does this require adding a root account in addition to the Administrator account

Reply:
No, Windows ignores the user setting